stdlib/public/Platform/android.modulemap

```diff
@@ -556,6 +556,14 @@ module _bits_sa_family_t [system] {
   header "bits/sa_family_t.h"
   export *
 }
+
+module _bits_sockaddr_storage [system] {
```

Comment:
Is this backwards compatible?

Reply:
With NDK 26? As I wrote above, "it still worked fine with NDK 26 too."